### PR TITLE
Label change

### PR DESCRIPTION
**Pull request by kahummer**
fix: #775

### CODE REVIEW

**Comment by kahummer** [comment]
@p-netm remove 
```
"BCC_GOAL_MEASURE": {
    "message": "จำนวนกิจกรรมการให้สุขศึกษา",
    "description": ""
  },
```

File: src/configs/strings/th.json [hunk 1]
```diff
@@ -651,10 +651,6 @@
     "message": "ให้สุขศึกษาในพื้นที่ปฏิบัติงานอย่างน้อย 1 ครั้ง",
     "description": ""
   },
-  "BCC_GOAL_MEASURE": {
-    "message": "จำนวนกิจกรรมการให้สุขศึกษา",
-    "description": ""
-  },
```
Comment (cKellyDesign) [inline src/configs/strings/th.json:657]: Can we keep this and ask if the translation needs to be updated? @kahummer

Comment (p-netm) [reply src/configs/strings/th.json:657]: ok, cool.

**Comment by Naima-Bashir** [comment]
image
Tested this. It works as expected. The text has been changed to :
Behaviour Change Communication
Measure: Number of BCC Activities Completed

Progress: 0 of 1 activit(y|ies) (0%)

**Pull request merged**